wallet/AndroidManifest.xml

```diff
@@ -201,19 +204,6 @@
             android:name="de.schildbach.wallet.ui.BlockInfoActivity"
             android:label="@string/block_info"
             android:theme="@style/My.Theme.ChildActivity" />
-        <activity
-            android:name="de.schildbach.wallet.ui.RestoreWalletActivity"
-            android:configChanges="orientation|keyboard|keyboardHidden"
-            android:theme="@style/My.Theme.Dialog">
-            <intent-filter android:label="@string/import_keys_intent_filter_action">
-                <action android:name="android.intent.action.VIEW" />
-
-                <category android:name="android.intent.category.DEFAULT" />
-
-                <data android:mimeType="application/x-darkcoin-wallet-backup" />
-                <data android:mimeType="application/x-dash-wallet-backup" />
-            </intent-filter>
-        </activity>
```

Comment:
In Dash Wallet, 21 users have had a crash from using this activity.  There were 48 crashes in the past 60 days with android 9, 10 and 11 with the latest version.  Users are using this function.  Was this activity causing any trouble with invites?